tests/app/run.sh

```diff
@@ -37,5 +37,6 @@ artifact_file=/tests/$(basename "$artifact_file")
 
 while read -r; do
     echo "running $REPLY"
-    EXPECTED_CONTAINERS="${root_dir}/data/docker"/expected.containers-delta.txt ARTIFACT_FILE="${artifact_file}" DATA="${root_dir}/data/docker" MODULE="${module}" "$REPLY" && echo -e "$REPLY \E[0;32mPASSED\E[0m"
+    EXPECTED_CONTAINERS="${root_dir}/data/docker"/expected.containers-delta.txt ARTIFACT_FILE="${artifact_file}" DATA="${root_dir}/data/docker" MODULE="${module}" "$REPLY" || exit 1
+    echo -e "\n$REPLY \E[0;32mPASSED\E[0m\n"
```

Comment:
Suggestion:
Colours are nice, but I would keep them in a separate variable, or maybe even a tiny lib? :)

Reply:
sure. added `common/functions.sh` as a global file included by the top level run.sh in tests.